code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,309 @@
+    Given the resource "{apiroot}/geofencing-subscriptions/v0.3/" as geofencing base-url                                                           
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+
+############################ Happy Path Scenarios ########################
+  @geofencing_subscriptions_01_Create_geofencing_subscription_for_a_device_sync
+   Scenario:  Create geofencing subscription (sync creation)
+    Given that subscriptions are created synchronously
+    And a valid subscription request body 
+    When the request "createSubscription" is sent 
+    Then the response code is 201
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_02_Create_geofencing_subscription_for_a_device_async
+   Scenario:  Create geofencing subscription (async creation)
+    Given that subscriptions are created asynchronously
+    And a valid subscription request body 
+    When the request "createSubscription" is sent 
+    Then the response code is 202
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body complies with the OAS schema at "#/components/schemas/SubscriptionAsync"
+
+  @geofencing_subscriptions_03_Operation_to_retrieve_list_of_subscriptions_when_no_records
+   Scenario: Get a list of subscriptions when no subscriptions available
+    Given a client without subscriptions created
+    When the request "retrieveGeofencingSubscriptionList" is sent 
+    Then the response code is 200
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And each item in the response body, if any, complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_04_Operation_to_retrieve_list_of_subscriptions
+   Scenario: Get a list of subscriptions  
+    Given a client with subscriptions created
+    When the request "retrieveGeofencingSubscriptionList" is sent 
+    Then the response code is 200
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body has an array of itenms and complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_05_Operation_to_retrieve_subscription_based_on_an_existing_subscription-id
+   Scenario: Get a subscription based on existing subscription-id.
+    Given the request body is not available and path parameter "subscriptionId" is set to the identifier of an existing subscription
```

Comment:
```suggestion
    Given the path parameter "subscriptionId" is set to the identifier of an existing subscription
```
As changed above, we no longer indicate that a body is not available for operations that do not allow a request body